### ISSUE TITLE
Issue with hourly chart if selecting shorter range

### ISSUE THREAD

**Issue by 0-5-0** [Akylas/oss-weather#286]
Under Providers, if I set "Number of hours in hourly forecast" to 24, because I'd like the hourly chart on the main page to essentially show me the current day (so I can see the bars/icons with less crowding) then it also changes the hourly chart inside the individual day views. 

Setting it to 24 hours as an example, when I press Monday (today) the hourly chart shows the same as in the hourly forecast on the main page but when I open Tuesday (tomorrow) then the graph for Tuesday's daily view doesn't load because it pulls the 24 hour setting from the same "Number of hours in hourly forecast".

If I set the "Number of hours in hourly forecast" to 72 then I can see the hourly charts for Monday, Tuesday and Wednesday again but the hourly chart on the main page becomes a bit crowded again.

**Comment by farfromrefug** [comment]
@0-5-0 ok i see what you want. Feels more like a new option. Number of hours in hourly forecast is a global setting changing the number of hours forecast "parsed" from the provider and thus used all along the app.
I will add a new parameter for the number of hours in the main view hourly chart

**Comment by 0-5-0** [comment]
Yes I figured it was likely a global setting, great if it can be done separately so the main page has its setting and the daily list has its setting. Thanks :)

**Comment by 0-5-0** [comment]
This works well in version 150, thanks :)
